modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -2017,10 +2020,15 @@ public static void writeIntCollection(DataOutput out, Collection<Integer> col) t
         try {
             InetAddress inetAddr = InetAddress.getByName(addr);
 
-            for (NetworkInterface itf : asIterable(NetworkInterface.getNetworkInterfaces()))
-                for (InetAddress itfAddr : asIterable(itf.getInetAddresses()))
+            for (NetworkInterface itf : asIterable(NetworkInterface.getNetworkInterfaces())) {
+                if (IGNORED_INTERFACE_NAMES.contains(itf.getName()))
+                    continue;
```

Comment:
Why we want to hide name for such interface? I don't see any of usage of this method
@vsisko It is really not used. I think we should hide it anywhere.
@nva Why we should change something without any reason?